cs/caching.texy

```diff
@@ -239,10 +239,20 @@ $cache->clean([
 
 A nakonec zmiňme parametr Cache::ALL, který smaže vše.
 
+
 Úložiště
 ========
 
-Kromě představeného úložiště `FileStorage` najdete v Nette Framework také [MemcachedStorage |api:Nette\Caching\Storages\MemcachedStorage] ukládající data na server `Memcached` a [MemoryStorage|api:Nette\Caching\Storages\MemoryStorage] pro ukládání do paměti s životností aktuálního požadavku.
+Kromě představeného úložiště `FileStorage` jsou v Nette připraveny tyto další uložište:
+|------------------------------------
+| Úložiště | Popis | Hromadné čtení |
+|------------------------------------
+| [FileStorage |api:Nette\Caching\Storages\FileStorage] | výchozí úložiště s ukládáním do souboru na disk | - |
+| [MemcachedStorage |api:Nette\Caching\Storages\MemcachedStorage] `Deprecated` | data se ukládají na server `Memcached` | - |
+| [NewMemcachedStorage |api:Nette\Caching\Storages\NewMemcachedStorage] | data se ukládají na server `Memcached` pomocí memcached extension | ano |
```

Comment:
Tohle je uplne stejne viz nahore, akorat tam neni to deprecated ;-) Oboje to vyuziva memcache extension.